cloudquery/sdk/transformers/openapi.py

```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
         return pa.string()
 
 
-def oapi_definition_to_columns(definition: Dict) -> List[Column]:
+def get_column_by_name(columns: List[Column], name: str) -> Column:
```

Comment:
```suggestion
def get_column_by_name(columns: List[Column], name: str) -> Optional[Column]:
```

(will require `from typing import Optional`)